pkg/updater/pubsub.go

```diff
@@ -214,10 +214,14 @@ func processNotification(paths map[gcs.Path][]string, n *pubsub.Notification) ([
 	base := path.Base(obj)
 	dur, ok := namedDurations[base]
 	if !ok { // Maybe it is a junit file, should finish soon
-		if !strings.HasPrefix(base, "junit") || !strings.HasSuffix(base, ".xml") {
+		switch {
+		case strings.HasPrefix(base, "junit") && strings.HasSuffix(base, ".xml"):
+			dur = 5 * time.Minute
+		case strings.HasSuffix(base, ".txt") && strings.Contains(obj, "directory/"):
```

Comment:
nit: Consider a comment for what the new case covers (I keep forgetting these are symlinks?)

Reply:
Done